### DISCUSSION TITLE
240319

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#56] [팀회의]
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항

## 어제 한 일
메인페이지 캘린더 ui 수정, 모달 캘린더 ui 완성
## 오늘 할 일
mock데이터로 데이터 받아와 보기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
ㅠㅠ

## 오늘 할 일
- api 테스트 및 적용
- 그룹 페이지 탭 기능

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
x
## 오늘 할 일
어제 못한 일
## 오늘의 일정 (코어 타임)
너무 힘들어서 코어타임 지나고 해도 될까요
## 멘토님 질문 사항
